src/prefect/server/schemas/actions.py

```diff
@@ -548,17 +548,41 @@ class FlowRunNotificationPolicyUpdate(ActionBaseModel):
 class ArtifactCreate(ActionBaseModel):
     """Data used by the Prefect REST API to create an artifact."""
 
-    key: Optional[str] = FieldFrom(schemas.core.Artifact)
-    type: Optional[str] = FieldFrom(schemas.core.Artifact)
-    data: Optional[Union[Dict[str, Any], Any]] = FieldFrom(schemas.core.Artifact)
-    metadata_: Optional[Dict[str, str]] = FieldFrom(schemas.core.Artifact)
-    flow_run_id: Optional[UUID] = FieldFrom(schemas.core.Artifact)
-    task_run_id: Optional[UUID] = FieldFrom(schemas.core.Artifact)
```

Comment:
Why do none of these use `FieldFrom` anymore?

Reply:
Good catch, this file should be left unchanged except for removing the `data` field in `ArtifactUpdate`